modules/saas/apps/saml-saas/src/main/java/com/liferay/saml/saas/internal/portlet/action/ExportSamlSaasMVCActionCommand.java

```diff
@@ -71,7 +71,7 @@
 	immediate = true,
 	property = {
 		"javax.portlet.name=" + SamlAdminPortletKeys.SAML_ADMIN,
-		"mvc.command.name=/admin/saas/saml/export"
+		"mvc.command.name=/admin/saml/saas/export"
```

Comment:
@csierra @martamedio I actually think we should not include "saml" in the name because then there is consistency with other MVCRenderCommands. The "saml" association is essentially provided by the portlet mapping.

Reply:
ah... ok then